### ISSUE TITLE
NullPointerException in CompletableProgressDialog

### ISSUE THREAD

**Issue by samtstern** [firebase/FirebaseUI-Android#921]
Reported by partner.

```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.ProgressBar.setVisibility(int)' on a null object reference
       at com.firebase.ui.auth.ui.phone.CompletableProgressDialog.onComplete(CompletableProgressDialog.java:63)
       at com.firebase.ui.auth.ui.phone.PhoneVerificationActivity.completeLoadingDialog(PhoneVerificationActivity.java:369)
       at com.firebase.ui.auth.ui.phone.PhoneVerificationActivity.onCodeSent(PhoneVerificationActivity.java:184)
       at com.firebase.ui.auth.ui.phone.PhoneVerificationActivity.access$700(PhoneVerificationActivity.java:55)
       at com.firebase.ui.auth.ui.phone.PhoneVerificationActivity$2.onCodeSent(PhoneVerificationActivity.java:256)
```

**Comment by TheCraftKid** [comment]
I thought we were getting rid of all the progress dialogs. :confused:

**Comment by samtstern** [comment]
This issue has been fixed in version `2.4.0`:
https://github.com/firebase/FirebaseUI-Android/releases/tag/2.4.0

**Comment by hackerutsav** [comment]
Hello,i have 2.3.0 and it's again coming here.It's not been fixed still :(
